### DISCUSSION TITLE
how to boot WIN10.ISO

### DISCUSSION THREAD

**Discussion by chainofhonor** [ipxe/ipxe#199] [Q&A]
sanboot http://192.168.1.1/WIN10.ISO
but,the WIN10.ISO is too big 

how to change the filesize limit?

**Comment by NiKiZe** [comment]
To be able to boot windows from iso you need real iSCSI target, not http.

But please checkout wimboot, you find documentation at iPXE.org

**Reply by Ridddick** [reply]
I have real iSCSI target and an iso image-file on it. How can i boot it? 
`Sanboot --filename \win10.iso ${iscsi-root} `
doesn't work.

**Reply by NiKiZe** [reply]
https://ipxe.org/cmd/sanboot gives a base
The iso have to be as an device, not on the filesystem on the device

Using your favourite search engine with ipxe sanboot should give even more.

**Reply by Ridddick** [reply]
In that case it's not an iso anymore! It's a disk image working just as common drive. That's what made me confused )) I just can't understand why it's posiible to boot a real iso via http but not via iscsi.

**Reply by NiKiZe** [reply]
it is not really possible to boot a iso over http either, explanations in the forum faq (do not be deceived by some parts of the boot process working)

And in boot cases - it as an image of a filesystem - no difference

**Reply by Hammarskjold** [reply]
Why do you want to live boot a Win10.iso? If you want to install Windows 10 you should boot the boot.wim and then deploy from there. Or what are you trying to achieve?
